templates/e-commerce/partials/blog-preview.html

```diff
@@ -0,0 +1,11 @@
+{{#articles}}
+<article class="md-pb7">
+  <amp-img class="mb3 md-col-10" src="{{images.0.narrow}}" srcset="{{images.0.narrow}} 750w,{{images.0.wide}} 1280w" width="{{imageWidth}}" height="{{imageHeight}}" layout="responsive" alt="{{title}}"></amp-img>
```

Comment:
Specifically `images.0.narrow` is used here because only the first image is needed to show the preview. 

This approach is used in the blog-article template. Only the first blog data is needed and all 3 images are mandatory to build the page. I could do some error checking to ensure the image exists before displaying it, would that solve this problem?